lib/identity-idp-functions.rb

```diff
@@ -6,6 +6,12 @@
 module IdentityIdpFunctions
   module_function
 
+  class MisconfiguredLambdaError < StandardError
+    def message
+      'IDP_API_AUTH_TOKEN is not configured'
+    end
+  end
+
```

Comment:
this file doesn't get loaded in the lambda world....we need to add shared things inside the `aws-ruby-sdk` folder and load it from `/opt/lib` in that weird way unforunately

Reply:
Since we load like 2 files that way, maybe we can a single file that `require_relative`s the others in?

Reply:
Ah, right. Moved the definition in d34d4fb127ac930f709cbc5d0d15fc9b924fb704

I'm not sure if I understand the second part. As in having `require IdentityIdpFunctions.helper_path('my one file')` with all the requires?

Reply:
What I meant was, every function has this preamble of requires:

```ruby
require 'bundler/setup' if !defined?(Bundler)
require 'json'
require 'retries'
require 'proofer'
require 'lexisnexis'
require '/opt/ruby/lib/faraday_helper' if !defined?(IdentityIdpFunctions::FaradayHelper)
require '/opt/ruby/lib/ssm_helper' if !defined?(IdentityIdpFunctions::SsmHelper)
```

So maybe we just define a single helper and it requires the others:

```ruby
require '/opt/ruby/lib/function_helpers' if !defined?(IdentityIdpFunctions::FunctionHelpers)
```

```ruby
# function_helpers.rb
require_relative './faraday_helper'
require_relative './ssm_helper'
require_relative './errors'
```

Reply:
Gave it a shot in b17fa66, does that look right?

Reply:
Yes! Big fan. Thank you!